presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -83,6 +83,18 @@ public void testDropTable()
     }
 
     @Test
+    public void testInsertIntoTable()
```

Comment:
It's better to put these in the `IntegrationSmokeTest` classes, since those are faster (they only run a few tests) and run every time.